app/models/post.rb

```diff
@@ -5,9 +5,15 @@ class Post < ApplicationRecord
   validates :title, length: { maximum: 250 }
   validates :comments_counter, numericality: { greater_than_or_equal_to: 0, only_integer: true }
```

Comment:
- [ ] It would be better if our current model and schema design can allow us to create posts and comments using the following commands
`   first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my first post') `
`   Comment.create(post: first_post, author: second_user, text: 'Hi Tom!' )  `
These commands are currently not working as we can see in the screenshot below

![Screenshot from 2023-02-03 00-54-10](https://user-images.githubusercontent.com/46388113/216484112-c9f9246e-765b-426d-9b20-abad10a00c76.png)